frontend/remote/nginx/nginx.conf

```diff
@@ -0,0 +1,7 @@
+server {
```

Comment:
I feel like it might be easier to just use the backend API to do this redirect no?

Reply:
Yeah maybe, since there's already nginx running in that container. Seems worth experimenting with after the launch.